ls1intum/Artemis
# [Feature] Embed Apollon-editor in modelling-exericse-update-component

### Checklist

### Motivation and Context

This change makes it easier for the instructors to create an Example Solution for a modelling exercise as discussed in this PR
https://github.com/ls1intum/Artemis/pull/3540#issuecomment-860722008

### Description

Before, the instructor could only create an Example Solution after creating a modelling exercise.
With the new changes,  we have embedded the Apollon editor inside the modelling-exercise-update-component, thus giving the instructor the possibility to create an Example Solution during the exercise creation/update. 

### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration
3. Select a course
4. Click exercises
5. Crete a modelling exercise
6. You should be able to create an example solution in the creation view of a modelling exercise.
6. Upon creation you should be redirected to the example submissions dashboard

### Test Coverage

Did not change.

### Screenshots and Gif
<img width="1224" alt="Screenshot 2021-06-20 at 7 32 56 PM" src="https://user-images.githubusercontent.com/51077603/122683457-9add6b80-d1ff-11eb-9cc8-aa8d2ea9e9eb.png">


![DEMO (modeling exercise update)](https://user-images.githubusercontent.com/51077603/122672033-c34a7300-d1c9-11eb-8e4b-60d7526bf56d.gif)
